doc/supportedPlatformVersions.md

```diff
@@ -1,347 +1,376 @@
 # Supported platforms and versions
 
-## dotnet
+## golang
```

Comment:
There is an active story for looking into the shuffling of this filehttps://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1664336

Reply:
Story for looking into the shuffling of this file: https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1664336